app/src/main/java/com/hedvig/app/feature/offer/ui/changestartdate/ChangeDateBottomSheetViewModel.kt

```diff
@@ -17,10 +21,14 @@ class ChangeDateBottomSheetViewModel(
 ) : ViewModel() {
 
     private val _viewState = MutableStateFlow<ViewState>(ViewState.Inceptions(data.inceptions))
-    val viewState: StateFlow<ViewState> = _viewState
+    val viewState: StateFlow<ViewState> = _viewState.asStateFlow()
```

Comment:
```suggestion
    val viewState = _viewState.asStateFlow()
```

Only one is really necessary 🙂